src/clj/game/cards-ice.clj

```diff
@@ -825,6 +825,27 @@
                  :cost [:credit 1] :effect (effect (add-prop target :advance-counter 1))}]
     :subroutines [(tag-trace 2)]}
 
+   "Mausolus"
+   {:advanceable :always
+    :subroutines [{:label "Gain 1 (Gain 3)"
+                   :msg (msg "gain " (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3) 1 3) " [Credit]")
+                   :effect (req (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3)
+                                  (gain state side :credit 1)
+                                  (gain state side :credit 3)))}
```

Comment:
We can reduce code repetition here by using `if` to select the amount of credits to gain in one `gain` statement:

`(gain state side :credit (if (< (+ ...) 3) 1 3))`

It would also let you use `(effect` instead of `(req` so you wouldn't need to pass `state side` manually.

`(effect (gain :credit (if (...) 1 3))`

Beautiful :).

Likewise for the `damage` calls in the second subroutine.